dev/PagerControl/PagerControl.xaml

```diff
@@ -1,28 +1,24 @@
-﻿<ResourceDictionary
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:MUXControlsTestApp"
-    xmlns:controls="using:Microsoft.UI.Xaml.Controls">
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls">
 
-    <x:Double x:Key="NumberPanelButtonWidth">32</x:Double>
-
-    <Style x:Key="NumberPanelButtonStyle" TargetType="Button">
-        <Setter Property="MinWidth" Value="{StaticResource NumberPanelButtonWidth}"/>
-        <Setter Property="MinHeight" Value="20"/>
-        <Setter Property="Background" Value="Transparent"/>
-    </Style>
-
-    <Style TargetType="local:PrototypePager">
+    <Style TargetType="muxc:PagerControl">
+        <Setter Property="AutomationProperties.LandmarkType" Value="Navigation"/>
+        <Setter Property="FirstButtonStyle" Value="{StaticResource NavigationBackButtonNormalStyle}"/>
```

Comment:
Can we add a dedicated PagerControl button style instead of using the globally defined NavigationBackButtonNormalStyle? That style is recommended [in the documentation](https://docs.microsoft.com/en-us/windows/uwp/design/basics/navigation-history-and-backwards-navigation#back-button) to be used as a button style for an in-app wide back button. It uses theme resources like `NavigationViewBackButtonBackground` so if customers have a PagerControl and an app wide navigate back button (for example in the titlebar) and override a resource like `NavigationViewBackButtonBackground` with the intention of changing the background of their in-app backwards navigation button, they will likely end up being surprised to see that the background of the PagerControl page navigate buttons have also changed.

I recommend creating a `PagerControlNavigationButtonStyle` instead, using PagerControl exclusive resources instead of resources like `NavigationViewBackButtonBackground` (if at all). Similar to the InfoBar control and its own dedicated `InfoBarCloseButtonStyle` for its close button:
```xml
    <Style x:Key="InfoBarCloseButtonStyle" TargetType="Button">
        <Style.Setters>
            <Setter Property="Width" Value="{StaticResource InfoBarCloseButtonSize}"/>
            <Setter Property="Height" Value="{StaticResource InfoBarCloseButtonSize}"/>
            <Setter Property="VerticalAlignment" Value="Top"/>
            <Setter Property="Background" Value="Transparent"/>
            <Setter Property="Margin" Value="4"/>
            <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}"/>
        </Style.Setters>
    </Style>
```

Reply:
I was also concerned about this, that's why it's one of the three open questions. I'll create a separate style.